onboarding/migrations/0025_auto_20210707_1943.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2 on 2021-07-07 19:26
+
+from django.db import migrations
+
+
+def change_other_housing_type_to_not_sure(apps, schema_editor):
+    OnboardingInfo = apps.get_model("onboarding", "OnboardingInfo")
+    print(len(OnboardingInfo.objects.filter(lease_type="OTHER")))
```

Comment:
Maybe remove this debugging `print()` call?

Reply:
ah, thought i caught em all but clearly not. thank you